src/module.c

```diff
@@ -7082,11 +7082,16 @@ int RM_Fork(RedisModuleForkDoneHandler cb, void *user_data) {
     return childpid;
 }
 
+/* Call from the child process when you want to send and update the cow info */
```

Comment:
please use:
```c
The module is advised to call this function from the fork child once in a while so that it can report COW memory to the parent which will be reported in INFO
```
@MeirShpilraien @yossigo @guybe7 maybe we wanna rename this API to be more generic?
i.e. maybe we'll find other usages for some heartbeat tasks in modules fork children that are optional?